src/main/java/seedu/fridgefriend/utilities/Logger.java

```diff
@@ -0,0 +1,32 @@
+package seedu.fridgefriend.utilities;
+
+import java.util.logging.Level;
+import java.lang.Throwable;
+
```

Comment:
Should explain what Logger class does here?

Reply:
will add javadoc comments